internal/ld/ld.go

```diff
@@ -114,13 +114,15 @@ func (c ApiClient) GetFlagKeyList(projKey string) ([]string, error) {
 	}
 
 	flagReq := c.ldClient.FeatureFlagsApi.GetFeatureFlags(ctx, projKey).Summary(true)
-	archiveReq := c.ldClient.FeatureFlagsApi.GetFeatureFlags(ctx, projKey).Summary(true).Archived(true)
+	archiveReq := c.ldClient.FeatureFlagsApi.GetFeatureFlags(ctx, projKey).Summary(true).Filter("state:archived")
+	deprecatedReq := c.ldClient.FeatureFlagsApi.GetFeatureFlags(ctx, projKey).Summary(true).Filter("state:deprecated")
```

Comment:
@cjhensen sounds like we're going to defer this change, so i'd like to keep things to two requests for now, if you don't mind changing it - still would love to keep the change of using `Filter` instead of `Archived`

Reply:
sure no problem, I'll update!

Reply:
thank you!